pep/app/pods/application/adapter.ts

```diff
@@ -82,6 +82,7 @@ export default class Application extends DS.RESTAdapter.extend(FastbootAdapter)
             const xForwardedFor = (fastbootHeaders.get('X-Forwarded-For') as string) ?? '';
             headers['X-Forwarded-For-PEP'] = xForwardedFor.split(',')[0];
         }
+        headers['x-api-authorize'] = 'pr2x%$ddE??233aGYKSE';
```

Comment:
i think this value should definitely be stored/injected  in our .env files, even if for now it doesnt change per environment, just to follow our usual patterns for organization of having all api-key like values in one place (and as something that has a decent chance of changing later, so even if its the same everywhere, having it injected .env at build-time means we dont need to do a code change to do so)

also if this is something thats added for every request, probably should be defined in the initial `const header` where `client-id` is set.

Reply:
Yea, this is totally temporary. Will figure out a different solution IF we keep it

Reply:
ok, as long as you aware Past-Adam will be held fully accountable for this if we end up leaving it and forgetting about it. 😛